lte/gateway/python/magma/enodebd/BUILD.bazel

```diff
@@ -39,6 +44,34 @@ py_binary(
     ],
 )
 
+py3_image(
+    name = "enodebd_image",
+    srcs = ["main.py"],
+    imports = [
+        LTE_ROOT,
+        ORC8R_ROOT,
+    ],
+    # legacy_create_init = False is required to fix issues in module import, see https://github.com/rules-proto-grpc/rules_proto_grpc/issues/145
+    legacy_create_init = False,
+    main = "main.py",
+    visibility = ["//visibility:public"],
+    deps = [
+        ":enodebd_lib",
+        "//lte/gateway/python/magma/enodebd/tr069:server",
+        "//lte/protos:mconfigs_python_proto",
+        "//orc8r/gateway/python/magma/common:sentry",
+        "//orc8r/gateway/python/magma/common:service",
+    ],
+)
```

Comment:
The arguments are the same as for the `py_binary` above, but without `python_version` because `py3_image` already sets that.